ts/features/pushNotifications/sagas/common.ts

```diff
@@ -69,10 +68,7 @@ export function* handlePendingMessageStateIfAllowed(
   // the application was killed and the push notification is tapped)
   yield* call(trackMessageNotificationTapIfNeeded, pendingMessageState);
 
-  // Check if there is a payment ongoing
-  const isPaymentOngoing = yield* select(isPaymentOngoingSelector);
```

Comment:
This check wouldn’t work as intended anyway, since the`isPaymentOngoingSelector` is scoped to the old payment flow. The new flow doesn’t currently use this selector, so it’s safe to remove it.